configuration/parser.go

```diff
@@ -206,6 +206,7 @@ func (p *Parser) overwriteStruct(v reflect.Value, fullpath string, path []string
 		fieldVal := reflect.New(sf.Type)
 		err := yaml.Unmarshal([]byte(payload), fieldVal.Interface())
 		if err != nil {
+			logrus.Warnf("Error parsing environment variable %s", fullpath)
 			return err
```

Comment:
Can we decorate this error message with more information maybe?
Maybe https://godoc.org/github.com/pkg/errors#Wrap ?

cc @stevvooe